RFC-by-stage/1-approved/update-grid.md

```diff
@@ -0,0 +1,26 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Update Grid 12 to use Flexbox
+
+## The issue to be solved
+
+Grid 12 is how the Design System does layout.
+
+Grid 12 was based on Bootstrap 3 (I think, it might be v4) and uses the box model to handle layout. Bootstrap v3 Grid docs here https://getbootstrap.com/docs/3.3/css/#grid and v4 here https://getbootstrap.com/docs/4.0/layout/grid/.
+
+However, Bootstrap is now at v5.1 and has moved to Flexbox, see https://getbootstrap.com/docs/5.1/layout/grid/, which is a good signal that we should too.
+
+There are increased benefits of using Flexbox - such as easier vertical centering, which would add capability to Grid 12.
```

Comment:
If we intend to continue maintaining & supporting Grid 12 internals (perhaps to offer an easy transition for people already relying on those classes in the old design system) flexbox sounds like a good idea 👍

But I wonder if we still need Grid 12 now CSS grid is more mature. Perhaps we could also consider deprecating Grid 12 internals and using CSS grid instead of flexbox?

Reply:
Yeah. 

The key reason for suggesting Flexbox is that it would be an easy port from Bootstrap. Although I agree, that CSS Grid seems more appropriate.